src/screens/ChatScreen/components/ChatList.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useRef } from 'react';
+import { FlatList, StyleSheet, View } from 'react-native';
+import { useSelector } from 'react-redux';
+
+import { ChatListItem } from 'app/screens/ChatScreen/components/ChatListItem';
```

Comment:
what do you think about using `index` file in main directories like `components` and import just from there instead of using path to destination?